lib/client-api/src/story_store.js

```diff
@@ -215,7 +215,7 @@ export default class StoryStore extends EventEmitter {
     return Object.keys(this._legacydata[key].stories)
       .map(name => this._legacydata[key].stories[name])
       .sort((info1, info2) => info1.index - info2.index)
-      .map(info => info.name);
+      .map(info => ({ name: info.name, parameters: info.parameters }));
```

Comment:
I have reviewed the codebase to ensure this getStories function isn't used outside of the callers updated in this PR, so don't consider this a breaking change.

Reply:
Wouldn't it be more beneficial at this point to just remove the map? You're basically just getting two properties and destructuring them into a new object. What more is there in these info objects?

Reply:
@Keraito thanks for the review.  The info objects also contain index and fn.  I chose to make the minimal possible change to what `getStories` returns.  You're right, index and fn could optionally be returned too, and callers could do with them what they please.  Let me know if that's the preference, and I'll update this PR.

Reply:
Personally, I don't mind too much, as either is fine. This also kinda influences my other comment, so maybe it's better to leave it at the changes you have made. Don't think the additional map has severe performance implications, but don't quote me on that 😆

Reply:
Thanks @Keraito - have pushed a commit that implements both of your suggestions.